### PR TITLE
Creating flake8.yml file

### PR DESCRIPTION
**Pull request by orIsrael**

### CODE REVIEW

File: .github/workflows/flake8.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    container: fedora:34
+    steps:
+      - name: Install Pipenv and Git
+        run: dnf install -y pipenv git
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup environment
+        run: pipenv sync --dev
+      - name: Run Flake8
+        run: pipenv run flake8 --max-line-length 120
```
Comment (yuvalk) [inline .github/workflows/flake8.yml:16]: no new line at end of file

**Comment by ShellyGolden** [comment]
@orIsrael I think that you need to squash the commits

File: .github/workflows/flake8.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
```
Comment (adiabramovitch) [inline .github/workflows/flake8.yml:17]: you can see that you are still missing a new line at the end of the file

**Comment by adiabramovitch** [comment]
You should squash the commits, three commits are not necessary. 
Use https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History to squash them. 
For the future, you can use ```git commit --amend``` if you want to add changes to your last commit.

**Comment by orIsrael** [comment]
I have fixed the problem you mentioned about the missing new line, however from some reason it doesn't appear in the PR

**Comment by adiabramovitch** [comment]
> I have fixed the problem you mentioned about the missing new line, however from some reason it doesn't appear in the PR

I have merged the commit but please add the additions needed in the Pipfile and Pipfile.lock

**Pull request merged**